providers/google/docs/changelog.rst

```diff
@@ -27,6 +27,37 @@
 Changelog
 ---------
 
+19.0.0
+......
```

Comment:
```suggestion
```

Reply:
One small suggestion - we are *just discussing the process of bumping version https://lists.apache.org/thread/2s15v9q9gy7llqxhqb3gnkvglcfb8ly8 -> and Idea is that the version number update is **always** done by the release manager or whoever is preparing the whole release docs (breaking changes in description should be added by the contributor though). Let's keep it this way for now.

cc: @eladkal ?

Reply:
Hello @potiuk Thank you for your review! I have a question regarding changelog.rst. Ok, I can leave the possibility to work with version number to the Release manager, but they should be aware about the changes. Basically my question is about where in the changelog.rst I should place notes about breaking changes?  Currently I add a new version number and place it under it. Or contributor just place it on top of the changelog.rst and RM will format doc during the release?

Reply:
It's even described in the comment above :) : 

<img width="1924" height="306" alt="Image" src="https://github.com/user-attachments/assets/cacc96e2-b07c-48f4-9a57-b59432831e7d" />